docs/UserGuide.md

```diff
@@ -223,7 +237,7 @@ Action | Format, Examples
 **Edit an event** | coming soon...
 **Add member to an event** | `enrol m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g.`enrol m/1 e/5 h/3 r/did planning`
 **Delete member from an event** | `unenrol m/MEMBER_INDEX e/EVENT_INDEX` <br> e.g.`unenrol m/1 e/5`
-**Edit an enrolment** | coming soon...
+**Edit an enrolment** | `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g. `editAttendance m/1 e/1 h/0 r/Absent due to Covid`
```

Comment:
Should we change this to `Edit an attendance` instead since the command is `editAttendance`?
```suggestion
**Edit an attendance** | `editAttendance m/MEMBER_INDEX e/EVENT_INDEX [h/NUMBER_OF_HOURS] [r/REMARK]` <br> e.g. `editAttendance m/1 e/1 h/0 r/Absent due to Covid`
```

Reply:
I think to make it consistent we should refactor all 'attendance` to `enrolment`.